src/styles/_sizes.media.scss

```diff
@@ -5,6 +5,42 @@ $medium-width: 1220px;
 $preview-small-width: 849px;
 $preview-medium-width: 1000px;
 
+$size-small-max: 374px;
+$size-medium-min: 375px;
+$size-medium-max: 767px;
+$size-large-min: 768px;
+$size-large-max: 1023px;
+$size-xlarge-min: 1024px;
+
+$SIZE_SMALL: '(max-width: #{$size-small-max})';
+$SIZE_MEDIUM: '(min-width: #{$size-medium-min}) and (max-width: #{$size-medium-max})';
+$SIZE_LARGE: '(min-width: #{$size-large-min}) and (max-width: #{$size-large-max})';
+$SIZE_XLARGE: '(min-width: #{$size-xlarge-min})';
+
+@mixin small-screen-size {
```

Comment:
Minor, but consider using a `breakpoint` mixin with the size as as string argument. Both Bootstrap and Foundation follow use a similar convention.

```
@mixin breakpoint($size)
```

Reply:
Added `breakpoint` mixin to _media-queries.scss file